enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+
+Reasons:
+
+* We can have tighter control on observability budget and spending
+* It’s our common, Red Hat responsibility to offer managed OpenShift. If Observability Group + AppSRE will not provide their support and time, the same (or more) effort and money has to be spent on OSD side (or vendor)
+* The monitoring Team already manages this service.
+  * We already work closely together with AppSRE, tightly integrated with them and familiar with AppInterface.
+  * We have devs on-call and SRE mindset.
+
+## Design Details
+
+We don’t want anyone to wait years to achieve this. As the Observability group, we want to give immediate value. Let’s see how we all can look when deployed. We proposed exact solutions that will have the required functionality, the quickest readiness and optimal stability and efficiency.
+
+In details it looks as below:
+
+![h](assets/hypershift-details.png)
+
+Let’s take a look at all parts:
+
+1. In terms of "Observability Product", we already have a growing open-source [Observatorium](http://observatorium.io/) project that packages and offers opinionated deployments of popular open-source projects for observability needs (e.g Thanos and Loki). What matters is that the Observability Group has a strong impact, deep knowledge and understanding of those projects. We also have ongoing production experience in running and scaling them.
+
+Anyone can install Observatorium, and deploy this service on-premise, but Observability Group also provides [RHOBS](https://rhobs-handbook.netlify.app/projects/observability/rhobs/) as a managed instance of observatorium, that is currently only available to specific internal teams.
+
+Observability Product will be a central pane of view of observability data for SRE needs from multiple clusters. Long term, it will also run critical Prometheus Alerting rules and alert routing for notification purposes.
+
+2. Since the Management cluster is like any other OCP/OSD, we can leverage Platform Monitoring. In near future, we plan to allow [MonitoringStack](https://github.com/openshift/enhancements/pull/866) project to allow scraping platform related metrics, but for the time being, this is done via Platform Monitoring that can remote write relevant metrics to Observability Product.
+
+3. [MonitoringStack](https://github.com/openshift/enhancements/pull/866) project offer similar functionality as UWM, but it aims for more features, flexibility and scalability due to different support models (outside of OpenShift Core) and less tight integration to OpenShift itself. It’s being developed in https://github.com/rhobs/monitoring-stack-operator. We plan to add support to [the agent mode of Prometheus](https://twitter.com/bwplotka/status/1454146102240944134) for efficient metric forwarding.
+4. A similar pipeline can be used for a portion of HSRE owned components living in each data plane.
```

Comment:
Please expand this. What change in the Hyprshift product is needed to get this done? How will the "similiar pipeline" know what is a HSRE owned component that it needs to scrape metrics from?